libconglomerate.user.js

```diff
@@ -1282,7 +1282,7 @@ function parseSafeItems() {
     safestatus = safe.snapshotLength;
     for (j = 0; j < safestatus; j++) {
         safeType = safe.snapshotItem(j).getAttribute('action').match(/op=([^"]*)/)[1];
-        safeOptions = document.evaluate('.//option', safe.snapshotItem(j), null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
+        safeOptions = document.evaluate('.//option[@value>="11"]', safe.snapshotItem(j), null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
```

Comment:
This has two assumptions that I wouldn't merge:
- that item IDs are never 10 or below
- that the amount of items you can bulk drop will never be increased above 10
If either of those is untrue/devs change it, then it becomes brittle code.

The original XPath query wasn't well defined--it's simply grabbing *any* option that is a descendent of the footlockergrab form. It was too generic.
Thankfully the devs differentiated the options into two different <select> tags:
- <select name='item'>
- <select name='discard_number'>

So I would suggest making the query more specific, i.e.
`'.//select[@name="item"]/option'`